lib/js/helpers/classy.js

```diff
@@ -14,7 +14,7 @@ export function has (domNode, className) {
 export function add (domNode, classes) {
   classes.split(' ').forEach(function (c) {
     if (!has(domNode, c)) {
-      var existingClass = domNode.getAttribute('class') || "";
+      var existingClass = domNode.getAttribute('class') || '';
```

Comment:
Your browser support is IE10 and up now, yes? http://caniuse.com/#search=classList